ChangeLog

```diff
@@ -15,6 +15,15 @@ Release date: TBA
 
   Closes #352
 
+* ``using-f-string-in-unsupported-version`` and ``using-final-decorator-in-unsupported-version`` msgids
```

Comment:
Hm, this is difficult. This is an important change, but this is technically breaking... Should it be in `3.0`?